tests/virt_autotest/reboot_and_wait_up_normal.pm

```diff
@@ -23,7 +23,7 @@ sub run {
     my $timeout = 180;
 
     #online upgrade actually
-    return if (is_remote_backend && check_var('ARCH', 'aarch64') && is_installed_equal_upgrade_major_release);
+    return if (is_remote_backend && check_var('ARCH', 'aarch64') && (is_installed_equal_upgrade_major_release || get_var("VIRT_PRJ1_GUEST_INSTALL") || get_var("VIRT_PRJ4_GUEST_UPGRADE")));
```

Comment:
After install or update pkg, a reboot is suggested to let them take effect. Why arm skips actual reboot?

Reply:
Because all arm64 openqa testsuites are being executed using SCC registration, all updates are retrieved and installed together with installation phase.

Reply:
For sle12sp4 at least, it skips SCC registration. Also AFAIK arm test do not install system_role related pkgs during host install. That's why you add a special PR for kvm_server/kvm_tools patterns in install_pkg.pm. Right?